src/StatusQ/Components/StatusClickableTag.qml

```diff
@@ -0,0 +1,139 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+/*!
+   \qmltype StatusAccountSelectorTag
+   \inherits Rectangle
+   \inqmlmodule StatusQ.Components
+   \since StatusQ.Components 0.1
+   \brief The StatusAccountSelectorTag provides a template for creating a clickable tag item.
+
+   Example of how to use it:
+
+   \qml
+        StatusClickableTag {
+            title: "some name"
+            icon.name: "some icon name"
+            isSelected: false
+            onClicked: {
+                // do something
+            }
+        }
+   \endqml
+
+   For a list of components available see StatusQ.
+*/
+Rectangle {
+    id: root
+
+    /*!
+       \qmlproperty titleText
+        This property represents the Text component which loads the text in the tag
+        Can be used to customise the Text component in the tag
+        \endqml
+    */
+    property alias titleText: titleText
+    /*!
+       \qmlproperty title
+        This property represents the title in the tag
+        Can be used to assign text to the tag
+        \endqml
+    */
+    property string title: ""
+    /*!
+       \qmlproperty isSelected
+        This property represents if the tag should be highlighted
+        \endqml
+    */
+    property  bool isSelected: false
+    /*!
+       \qmlproperty maxTextWidth
+        This property represents the max widht allowed for the title after which it will elide
+        \endqml
+    */
+    property int maxTextWidth: 93
+    /*!
+       \qmlproperty image
+        This property represents the StatusImageSettings to be shown in the tag
+        Can be used to assign image to the tag
+        \endqml
+    */
+    property StatusImageSettings image: StatusImageSettings {
+        width: 20
+        height: 20
+        isIdenticon: false
+    }
+    /*!
+       \qmlproperty icon
+        This property represents the StatusIconSettings to be shown in the tag
+        Can be used to assign icon to the tag
+        \endqml
+    */
+    property StatusIconSettings icon: StatusIconSettings {
+        height: 20
+        width: 20
+        rotation: 0
+        isLetterIdenticon: false
+        letterSize: 10
+        color: Theme.palette.primaryColor1
+        background: StatusIconBackgroundSettings {
+            width: 15
+            height: 15
+            color: Theme.palette.primaryColor3
+        }
+    }
+
+    signal clicked()
+
+    width: layout.width
+    height: 32
+    color: Theme.palette.statusAppLayout.backgroundColor
+    radius: 8
+
+    Rectangle {
+        anchors.fill: parent
+        color: isSelected ? Theme.palette.primaryColor2 :"transparent"
```

Comment:
This should have some hover indication as well

Reply:
One other thing I've noticed as well: The colors generally don't seem to match with the ones in the design. At least for the labels, probably also the background?

Reply:
so the thing is I think the wallet designs are syncing 100% with other designs and I think it expected it will be synced once the component design system that David is working on will be completed